src/ngraph/frontend/onnx_import/op/expand.cpp

```diff
@@ -35,11 +35,8 @@ namespace ngraph
                     const std::shared_ptr<ngraph::Node> data{node.get_ng_inputs().at(0)};
                     const std::shared_ptr<ngraph::Node> shape{node.get_ng_inputs().at(1)};
 
-                    const auto const_filled_with_ones = std::make_shared<default_opset::Broadcast>(
-                        default_opset::Constant::create(data->get_element_type(), {}, {1}), shape);
-
-                    return {
-                        std::make_shared<default_opset::Multiply>(data, const_filled_with_ones)};
+                    return {std::make_shared<default_opset::Broadcast>(
```

Comment:
`Broadcast` from opset other than `opset3` would not work. Does it mean we have to have some if-else logic based on the opset version?

Reply:
`default_opset` is just an alias that we use for onnx_importer. In this PR it was switched to point to opset3 https://github.com/NervanaSystems/ngraph/pull/4655/files?file-filters%5B%5D=.hpp#diff-1ca2b8cb1f7660163e9568e1ae2f2601R7

Reply:
I get it, but should we have a comment here or smth that points out difference between opsets? What if someone change default_opset and forget about Broadcasts?

Reply:
I don't think so TBH. The `default_opset` alias should always point to the most recent ngraph-core opset. It was introduced so that all onnx_importer files reference the core ops in an uniform way. If and only if there's a need to reference any of the previous opsets in a particular ONNX op implementation, the developer should use `opsetX::OpName` explicitly.

The alias should not be changed by anyone but our team. If we switch to `opset4` at some point, this line we're discussing now will automatically use `Broadcast` from `opset4`. If this will be a new version of the operator and the constructor changes its signature, the person updating the alias will have to correct the usage and fix all of the compilation errors (and tests).